docs/content/configuration/csi_ug_config_advanced_importvol.md

```diff
@@ -37,128 +37,121 @@ To find the `volumeHandle`, use one of the following procedures:
 
       The **Volume Properties** form appears.
 
-  3.  Use the **ID** number.
+  3. Use the **ID** number.
     
       For more information, see [IBM Hyper-Scale Manager documentation](https://www.ibm.com/docs/en/hyper-scale-manager/).
 
 
-Use this procedure to help build a PV yaml file for your volumes.
+Use this procedure to help build a PV YAML file for your volumes.
 
 **Note:** These steps are setup for importing volumes from a Spectrum Virtualize Family system. Change parameters, as needed.
 
-1.  Create a persistent volume (PV) yaml file.
+1. Create a persistent volume (PV) YAML file.
 
     **Important:** Be sure to include the `storageClassName` and `controllerPublishSecretRef` parameters or errors will occur.
 
-2.  Take the `volume_name` and other optional information (collected before the procedure) and insert it into the yaml file.
-
-    <pre>
-    apiVersion: v1
-    kind: PersistentVolume
-    metadata:
-      # annotations:
-        # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
-      name: vol1-pv
-    spec:
-      accessModes:
-      - ReadWriteOnce
-      capacity:
-        storage: 1Gi
-      csi:
-        controllerPublishSecretRef:
-          name: demo-secret
-          namespace: default
-        driver: block.csi.ibm.com
-        <b># volumeAttributes:
-          # pool_name: ibmc-block-gold
-          # storage_type: SVC
-          # volume_name: vol1
-          # array_address: baremetal10-cluster.xiv.ibm.com
-        volumeHandle: SVC:600507640082000B08000000000004FF</b>
-      # persistentVolumeReclaimPolicy: Retain
-      storageClassName: ibmc-block-gold
-      # volumeMode: Filesystem
-    </pre>
-
-3.  Create a PersistentVolumeClaim (PVC) yaml file.
+2. Take the `volume_name` and other optional information (collected before the procedure) and insert it into the YAML file (under `spec.csi.volumeAttributes`).
+
+    **Important:** If using the CSI Topology feature, the `spec.csi.volumeHandle` contains the system ID. In the example below, the `spec.csi.volumeHandle` would read similar to the following: `SVC:demo-system-id-1:600507640082000B08000000000004FF`.
+    
+        apiVersion: v1
+        kind: PersistentVolume
+        metadata:
+          # annotations:
+            # pv.kubernetes.io/provisioned-by: block.csi.ibm.com
+          name: vol1-pv
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          capacity:
+            storage: 1Gi
+          csi:
+            controllerPublishSecretRef:
+              name: demo-secret
+              namespace: default
+            driver: block.csi.ibm.com
+            # volumeAttributes:
+              # pool_name: ibmc-block-gold
+              # storage_type: SVC
+              # volume_name: vol1
+              # array_address: baremetal10-cluster.xiv.ibm.com
+            volumeHandle: SVC:600507640082000B08000000000004FF
+          # persistentVolumeReclaimPolicy: Retain
+          storageClassName: ibmc-block-gold
```

Comment:
```suggestion
              # pool_name: demo-pool
              # storage_type: SVC
              # volume_name: demo-prefix_demo-pvc-file-system
              # array_address: demo-management-address
            volumeHandle: SVC:600507640082000B08000000000004FF
          # persistentVolumeReclaimPolicy: Retain
          storageClassName: demo-storageclass
```

Reply:
@barakda  are the changes suggested above OK with you as well before I make the changes?

Reply:
I think @roysahar-ibm is the one who handled with volume import

Reply:
@oriyarde I did not run the procedure manually as described in the guide. @barakda did we run this manually as described in the guide?

Reply:
@barakda ?

Reply:
it looks okay (also reviewed by Tal who has tested the import during the last version) - approved

Reply:
@oriyarde  please resolve

Reply:
@barakda by "it" you mean my suggestion or current content?
because I don't see my suggestion in current diff

Reply:
oh sorry, with your suggestion of course

Reply:
thanks, @barakda 
DONE